src/modules/layouts/SiteContext.js

```diff
@@ -117,6 +117,8 @@ const SiteProvider = ({ children, value }) => {
     )
   }, {})
 
+  console.log(AllData);
```

Comment:
Suggestion to ✂️ this if it's retained in prod!

Reply:
Ah, awesome catch thank you!